soccer/src/soccer/strategy/coach/coach_node.hpp

```diff
@@ -38,7 +40,14 @@ class CoachNode : public rclcpp::Node {
 
 private:
     rclcpp::Publisher<rj_msgs::msg::CoachState>::SharedPtr coach_state_pub_;
+
+    rclcpp::Publisher<rj_geometry_msgs::msg::ShapeSet>::SharedPtr def_area_obstacles_pub_;
+    rclcpp::Publisher<rj_geometry_msgs::msg::ShapeSet>::SharedPtr global_obstacles_pub_;
+
+    rclcpp::Subscription<rj_msgs::msg::FieldDimensions>::SharedPtr field_dimensions_sub_;
+
     rclcpp::Subscription<rj_msgs::msg::PlayState>::SharedPtr play_state_sub_;
+
```

Comment:
Could you add a doc string for each of the pub/subs so that its documented why they're important.